software/spmd/bsg_cuda_lite_runtime/conv2d/Makefile

```diff
@@ -0,0 +1,47 @@
+#########################################################
+# Network Configutaion
+# If not configured, Will use default Values
+	bsg_global_X ?= $(bsg_tiles_X)
+	bsg_global_Y ?= $(bsg_tiles_Y)+1
+
+#########################################################
+#Tile group configuration
+# If not configured, Will use default Values
+	bsg_tiles_org_X ?= 0
+	bsg_tiles_org_Y ?= 1
+
+# If not configured, Will use default Values
+	bsg_tiles_X ?= 2
+	bsg_tiles_Y ?= 2
+
+
+all: main.run
+
+# Rule to write processor execution logs. To be used after the
+# verilog simulation.
+#
+# Redirects verilog standard output starting with "X<x_cord>_Y<y_cord>.pelog" 
+# to a unique log file for each coordinate in the manycore. This can be useful 
+# for a quick debug of processor or program running on it.
+proc_exe_logs: X0_Y0.pelog X1_Y0.pelog
+
```

Comment:
These rules are no longer used. please stop copying and pasting these.

Reply:
Then they should be removed from every other spmd test too right?

Reply:
That's right.

Reply:
Hi Sasha, it would be great if you can remove them from this test, as it seems you have done. 

File an issue for the remaining ones -- it's not your responsibility to maintain the manycore repository.